scripts/quests/jeuno/Shadows_of_the_Departed.lua

```diff
@@ -0,0 +1,143 @@
+-----------------------------------
+-- Shadows of the Departed
+-----------------------------------
+-- !addquest 3 88
+-- Ru'Lude Gardens Palace Region 1
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/common')
+require('scripts/globals/items')
+require('scripts/globals/keyitems')
+require("scripts/globals/npc_util")
+require('scripts/globals/quests')
+require('scripts/globals/titles')
+require('scripts/globals/zone')
+-----------------------------------
+local ID = require('scripts/zones/RuLude_Gardens/IDs')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.SHADOWS_OF_THE_DEPARTED)
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:hasCompletedQuest(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.STORMS_OF_FATE) and
+            player:getCurrentMission(xi.mission.log_id.ZILART) == xi.mission.id.zilart.AWAKENING and
+            player:getMissionStatus(xi.mission.log_id.ZILART) == 3 and
+            player:getQuestStatus(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.SHADOWS_OF_THE_DEPARTED) == QUEST_AVAILABLE and
+            player:getCharVar("StormsOfFateWait") <= os.time()
+        end,
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            onTriggerAreaEnter =
+            {
+                [1] = function(player, region)
+                    player:startEvent(161)
```

Comment:
do we use player:startEvent here?

Reply:
I think because we have not started the quest yet?

Reply:
we should still be returning a quest object. So perhaps let's do:
```suggestion
                    return quest:progressEvent(161)
```